### ISSUE TITLE
Make icon color theme dependant for black and/or white icons

### ISSUE THREAD

**Issue by Snailedlt** [Snailedlt/Snailedlt#17]
How to -> https://www.stefanjudis.com/notes/how-to-define-dark-light-mode-images-in-github-markdown/
Relevant repo with snippets -> https://github.com/stefanjudis/github-light-dark-image-example/blob/main/README.md

Like what I did for the GitHub icon -> https://github.com/Snailedlt/Snailedlt/commit/9c404a2c45173ac5b379b9c40d1051a472b95899

**Comment by MrSarno** [comment]
Was about to suggest this; the Unity icon is pretty difficult to see if using the ~~best~~ dark default theme.

**Comment by Snailedlt** [comment]
@MrSarno Thanks for the concrete example :)
Yeah agreed! Will fix this in the near future.

**on a side note:** I have no idea why you striked out `best`... because it clearly is!

**Comment by MrSarno** [comment]
@Snailedlt I agree, but I figured "the best theme" was not very descriptive. :P
